app/facades/tutorial_facade.rb

```diff
@@ -17,7 +17,7 @@ def next_video
   end
 
   def play_next_video?
-    (current_video.position < maximum_video_position)
+    (current_video.position < maximum_video_position.position)
```

Comment:
This was something from the original that didn't work, didn't break it when we were doing rubocop checks